lib/Facebook.js

```diff
@@ -815,10 +816,274 @@ function Facebookbot(configuration) {
+
+            var body = {
+                'user' : user
+            };
+
+            request({
+                method: 'POST',
+                json: true,
+                body: body,
+                uri: uri
+            }, function(err, res, body) {
+                if (err) {
+                    facebook_botkit.log('Could not add a user to a label');
+                    cb(err);
+                } else {
+                    if (body.error) {
+                        facebook_botkit.log('ERROR in adding a user to a label API call : ', body.error);
+                        cb(body.error);
+                    } else {
+                        facebook_botkit.debug('Successfully add a user to a label', body);
+                        cb(null, body);
+                    }
+                }
+            });
+        },
+        remove_user_from_label: function(user, label_id, cb) {
+            var uri = 'https://' + api_host + '/' + api_version + '/' + label_id + '/label?access_token=' + configuration.access_token;
+
+            var body = {
+                'user' : user
+            };
+
+            request({
+                method: 'DELETE',
+                json: true,
+                body: body,
+                uri: uri
+            }, function(err, res, body) {
+                if (err) {
+                    facebook_botkit.log('Could not remove a user from a label');
+                    cb(err);
+                } else {
+                    if (body.error) {
+                        facebook_botkit.log('ERROR in removing a user from a label API call : ', body.error);
+                        cb(body.error);
+                    } else {
+                        facebook_botkit.debug('Successfully remove a user from a label', body);
+                        cb(null, body);
+                    }
+                }
+            });
+        },
+        get_labels_by_user: function(user, cb) {
+            var uri = 'https://' + api_host + '/' + api_version + '/' + user + '/custom_labels?access_token=' + configuration.access_token;
 
+            request({
+                method: 'GET',
+                json: true,
+                uri: uri
+            }, function(err, res, body) {
+                if (err) {
+                    facebook_botkit.log('Could not get user associated labels');
+                    cb(err);
+                } else {
+                    if (body.error) {
+                        facebook_botkit.log('ERROR in getting user associated labels API call : ', body.error.message);
+                        cb(body.error);
+                    } else {
+                        facebook_botkit.log('Successfully get user associated labels');
+                        cb(null, body);
+                    }
+                }
+            });
+        },
+        get_label_details: function(label_id, fields, cb) {
```

Comment:
Currently the only available extra field for a label is `name`, which I think we can just add as the default to this request and save the additional argument to get_label_details

Reply:
I put fields to prevents any futur break changes, Exactly, today, name is the only available field, bu FB can add some other ones.